services/node/node-current.service.js

```diff
@@ -19,4 +19,8 @@ module.exports = class NodeCurrentVersion extends NodeVersionBase {
   static get colorResolver() {
     return versionColorForRangeCurrent
   }
+
+  static get documentation() {
+    return `The badge will be green if the selected package release supports the current (latest) node version`
```

Comment:
I'd caution against trying to describe the colors within the description (also FWIW the color is `brightgreen`) and instead describe the information that the badge conveys.

The intent of this badge is to allow users (presumably package owners) to convey the status of support their package has for the current/latest node release, and conveying that message with whatever wording makes sense (independent of color mapping) should suffice.